scripts/changelog/assemble_changelog.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import git
+
+
+def get_changes(path):
```

Comment:
What is the testing strategy for the changelog logic? Do I understand correctly that after each change I need to manually run script locally simulating different scenarios that my change may affect? Do you think it makes sense to add auto tests, so that developers could enhance this logic without being afraid break a different scenario?

I think this question isn't a blocker for this particular PR, but it's worth to discuss 🙂

Reply:
yeah, good point, let's add tests for this logic